configure.ac

```diff
@@ -1024,7 +1024,10 @@ AC_ARG_ENABLE([tests],
               [HDF5_TESTS=$enableval])
 
 if test "X$HDF5_TESTS" = "Xno"; then
+  AC_MSG_RESULT([yes])
   echo "Building HDF5 tests is disabled"
+else
+  AC_MSG_RESULT([no])
 fi
 
```

Comment:
Disabling tools and tests doesn't seem to have any relation to doxygen, so I wondered why it's in this PR?  The changes match what is in develop, so I suppose they're ok.
I also find it confusing that "Xno" results in  "yes", but I guess it's because the question was negative?

Reply:
They are here because without them the stdout  munges all three configure text on one line.
And yes the question formats the answer.